lib/fog/storage/google_json/models/file.rb

```diff
@@ -60,7 +87,6 @@ def public_url
         FILE_INSERTABLE_FIELDS = %i(
           content_type
           predefined_acl
-          acl
```

Comment:
Why deleting this line?

Reply:
if i pass parameter "acl" in "fog_options". I get error: "undefined method `each_with_index' for "publicRead":String", seems acl is not a allowed parameter. 

```
representable (3.0.4) lib/representable/pipeline.rb:38:in `call'
representable (3.0.4) lib/representable/pipeline.rb:18:in `evaluate'
representable (3.0.4) lib/representable/pipeline.rb:10:in `block in call'
representable (3.0.4) lib/representable/pipeline.rb:9:in `each'
representable (3.0.4) lib/representable/pipeline.rb:9:in `inject'
representable (3.0.4) lib/representable/pipeline.rb:9:in `call'
representable (3.0.4) lib/representable/binding.rb:32:in `compile_fragment'
representable (3.0.4) lib/representable.rb:53:in `block in call'
representable (3.0.4) lib/representable.rb:51:in `each'
representable (3.0.4) lib/representable.rb:51:in `call'
representable (3.0.4) lib/representable.rb:70:in `representable_map!'
representable (3.0.4) lib/representable.rb:45:in `create_representation_with'
representable (3.0.4) lib/representable/hash.rb:34:in `to_hash'
representable (3.0.4) lib/representable/json.rb:44:in `to_json'
google-api-client (0.19.8) lib/google/apis/core/api_command.rb:64:in `prepare!'
google-api-client (0.19.8) lib/google/apis/core/upload.rb:51:in `prepare!'
google-api-client (0.19.8) lib/google/apis/core/upload.rb:141:in `prepare!'
google-api-client (0.19.8) lib/google/apis/core/http_command.rb:91:in `execute'
google-api-client (0.19.8) lib/google/apis/core/base_service.rb:360:in `execute_or_queue_command'
google-api-client (0.19.8) generated/google/apis/storage_v1/service.rb:1981:in `insert_object'
fog-google (1.3.3) lib/fog/storage/google_json/requests/put_object.rb:66:in `put_object'
fog-google (1.3.3) lib/fog/storage/google_json/models/file.rb:78:in `save'
fog-core (2.1.0) lib/fog/core/collection.rb:50:in `create'
paperclip (5.1.0) lib/paperclip/storage/fog.rb:112:in `block in flush_writes'
paperclip (5.1.0) lib/paperclip/storage/fog.rb:101:in `each'
paperclip (5.1.0) lib/paperclip/storage/fog.rb:101:in `flush_writes'
paperclip (5.1.0) lib/paperclip/attachment.rb:244:in `save'
```

Reply:
And "predefined_acl" is work.